app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    path = command.split(" ")
```

Comment:
```suggestion
    path = command.split()
```
Also `path` not clear name